src/monitor_endpoint.py

```diff
@@ -82,7 +99,21 @@ def monitor_info():
     for balance in exchange_balances or []:
         info[f'exchange_{balance.symbol}_available'] = float(format(balance.available, f'.{balance.decimals}f'))
         info[f'exchange_{balance.symbol}_total'] = float(format(balance.total, f'.{balance.decimals}f'))
+        if balance.symbol == 'NZD':
+            if balance.available < EXCHANGE_MIN['NZD']:
+                info['exchange_NZD_available_threshold_count'] = 1
+            else:
+                info['exchange_NZD_available_threshold_count'] = 0
+        if balance.symbol == 'BTC':
+            if balance.available < EXCHANGE_MIN['BTC']:
+                info['exchange_BTC_available_threshold_count'] = 1
+            else:
+                info['exchange_BTC_available_threshold_count'] = 0
```

Comment:
```suggestion
        if balance.symbol in EXCHANGE_MIN:
            if balance.available < EXCHANGE_MIN[balance.symbol]:
                info[f'exchange_{balance.symbol}_below_min'] = 1
            else:
                info[f'exchange_{balance.symbol}_below_min'] = 0
```

dont hardcode accesses to `EXCHANGE_MIN` that could crash if the default value changed (and simplify code a bit)